allolib-wasm/include/al_WebGPUCollision.hpp

```
@@ -0,0 +1,361 @@
+        if (index < 0 || index >= (int)mParams.colliderCount) return;
+        mParams.colliders[index].friction = f;
+    }
+
+    /// Set restitution for a collider
+    void setRestitution(int index, float r) {
+        if (index < 0 || index >= (int)mParams.colliderCount) return;
+        mParams.colliders[index].restitution = r;
+    }
+
+    /// Apply collisions to the particle system
+    void apply(ParticleSystem& ps, float dt) {
```

Clang (14.0.6)

[warning] method 'apply' can be made static

(readability-convert-member-functions-to-static)